src/tests/setup.ts

```diff
@@ -12,9 +12,11 @@ export default async (): Promise<void> => {
     process.env.NODE_ENV = 'test';
   }
   loadDotEnv();
+  process.env.PG_DATABASE = 'postgres';
+  const db = await PgDataStore.connect({ skipMigrations: true, usageName: 'setup' });
   const server = await startEventServer({
```

Comment:
If I'm not mistaken, this entire global server is never used... Can you try removing it (also removing the db) from here and `teardown.ts` to see if tests are at all affected? If not, we should delete it.